apps/wallet-mobile/src/InitApp.tsx

```diff
@@ -51,11 +52,14 @@ const initInstallationId = async (storage: App.Storage) => {
 
   const newInstallationId = uuid.v4()
   await storage.setItem('appSettings/installationId', newInstallationId, () => newInstallationId) // LEGACY: installationId is not serialized
+
+  // new installation set the storage version to the current version
+  await storageVersionMaker(storage).newInstallation()
```

Comment:
new installations set the storage version to the last version (no need of migrations) obviously.